modules/apps/web-experience/site-navigation/site-navigation-breadcrumb-web/src/main/resources/com/liferay/site/navigation/breadcrumb/web/portlet/template/dependencies/portlet_display_template_arrows.ftl

```diff
@@ -34,21 +43,40 @@
 				width: 0;
 			}
 
-			.breadcrumb-arrows a:first-child {
+			.breadcrumb-arrows li:first-child > a {
 				-moz-border-radius: 4px 0 0 4px;
 				-webkit-border-radius: 4px 0 0 4px;
```

Comment:
I think we can remove these browser prefixes since they are only needed for the older mozilla and chrome browsers which we are no longer supporting in liferay-portal/master.